sonic_y_cable/y_cable.py

```diff
@@ -1401,3 +1408,84 @@ def rollback_firmware(physical_port):
     """
 
     return FIRMWARE_ROLLBACK_SUCCESS
+
+
+def enable_auto_switch(physical_port, mode):
```

Comment:
Wondering whether we should call this something like `set_switching_mode()` and pass in a constant like `SWITCHING_MODE_AUTO` or `SWITCHING_MODE_MANUAL`. We would then change `check_if_auto_switch_enabled()` to `get_switching_mode()` and it would return one of these constants.

Reply:
fixed